02-peer-protocol.md

```diff
@@ -11,9 +11,25 @@ The messages described in this document are grouped logically into 4 groups by t
  - Routing (types `256`-`511`): node and channel announcements, as well as any active route exploration.
 
 # Table of Contents
-
-FIXME
-
+  * [Channel](#channel)
```

Comment:
Hmm, there's shutdown, which is where we don't add anything new, and closing, which is where we're negotiation the final close tx.  I used to call shutdown "clearing" but that has a specific finance meaning which was deeply sucky.

Ah, I see you reverted that AFAICT?  I'll squash and merge.